tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-tax-calculator-test.php

```
@@ -0,0 +1,426 @@
+		);
+
+		$event      = $this->build_event( [ $this->product, $product2 ] );
+		$line_items = $this->calculator->extract_line_items_from_customization_hook( $event );
+		$result     = $this->calculator->calculate( $event, $line_items );
+
+		$this->assertCount( 2, $result['line_items'] );
+		$this->assertNotEmpty( $result['line_items'][0]['tax_rates'] );
+		$this->assertNotEmpty( $result['line_items'][1]['tax_rates'] );
+
+		$product2->delete( true );
+	}
```

PHPMD (2.15.0)

[error] The method test_calculate_with_multiple_products is not named in camelCase. (undefined)

(CamelCaseMethodName)